server/src/main/java/io/crate/expression/reference/doc/lucene/SourceParser.java

```diff
@@ -151,9 +168,19 @@ private static Map<String, Object> parseObject(XContentParser parser,
             XContentParser.Token token = parser.nextToken(); // move past START_OBJECT;
             for (; token == XContentParser.Token.FIELD_NAME; token = parser.nextToken()) {
                 String fieldName = parser.currentName();
+                boolean dropped = false;
+                if (droppedColumns.isEmpty() == false) {
+                    String path = "";
+                    if (colPath.isEmpty() == false) {
+                        path = colPath + ".";
+                    }
+                    path += fieldName;
```

Comment:
minor (or maybe not for `select _raw from big_table` without any filters) - can we use `StringBuilder` for path as well?

Reply:
hm, also I forgot about this, currently this doesn't work, as for `_raw` we don't use `SourceParser` at all, we directly return the source as is: `SourceLookup#rawSource()`,
I will add a point to the issue about remaining tasks.

Reply:
I see, this looks out of the scope of this PR. Sorry, I just misunderstood and thought SourceParser changes already cover this.